app/models/meetup.rb

```diff
@@ -0,0 +1,10 @@
+class Meetup < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :court
+
+  validates_presence_of :start_time
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.